<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1828</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="MotorSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>501</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Bearing test settings</string>
    </property>
    <widget class="QSplitter" name="splitter_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>460</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Revolutions/Direction</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="motor_revolutions_per_direction_spinBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>99999999</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>460</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Revolutions</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="motor_total_revolutions_spinBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>160000</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>460</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pause time (sec)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="pause_time_spinBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>1300000</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>460</width>
       <height>161</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Motor Speed (rev/min)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="motor_speed_spinBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Motor Acceleration (rev/min^2)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="motor_acceleration_spinBox">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="motor_anti_CW_radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>460</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Turn &amp;motor anti-CW only</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="motor_CW_radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>460</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Turn motor CW only</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="motor_alternate_direction_radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>460</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Alternate CW and anti-CW</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>1281</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Capture settings</string>
    </property>
    <widget class="QSlider" name="led_intensity_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>400</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Camera Exp. time (ms)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="camera_exp_time_spinBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>400</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LED 1 Intensity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Video Duration (sec)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="video_duration_spinBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="led_intensity_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>400</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>400</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>LED 2 Intensity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="image_camera_livestream">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>651</width>
       <height>421</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>CAMERA image</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CaptureImagesDuringRotation">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>361</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string>Capture images during rotation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>203</y>
       <width>400</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create Video every N cycles</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="video_cycle_spinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>400</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Bearing test control and progress</string>
    </property>
    <widget class="QPushButton" name="start_run_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>461</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start Run</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>461</width>
       <height>91</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Progress</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="revolutions_counter">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="doCCWindex">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>CCW index</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doCWindex">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>CW index</string>
     </property>
    </widget>
    <widget class="QLabel" name="encoderAValue">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="encoderBValue">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>1281</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Monitoring</string>
    </property>
    <widget class="QChartView" name="TorqueChart" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>1261</width>
       <height>351</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>451</width>
       <height>67</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Temperature 1 (K)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="temp_sensor_1_lcdNumber">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Temperature 2 (K)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="temp_sensor_2_lcdNumber">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Temperature 3 (K)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="temp_sensor_3_lcdNumber">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter_10">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>400</y>
       <width>400</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sampling time (ms)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sampling_time">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="microseconds_current">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>420</y>
       <width>211</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1828</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
